encoders/numeric/FastICAEncoder/README.md

```diff
@@ -0,0 +1,6 @@
+# FastICAEncoder
+
+`FastICAEncoder` encodes data from an ndarray in size `B x T` into an ndarray in size `B x D`.
+
+note:
```

Comment:
that is wrong under current level, that was before, now u can pass a pickled file from which to do encoding